src/main/webapp/app/course/tutorial-groups/shared/tutorial-group-detail/tutorial-group-detail.component.html

```diff
@@ -38,6 +38,37 @@ <h3>{{ 'artemisApp.pages.tutorialGroupDetail.title' | artemisTranslate }}:</h3>
                 <span>{{ 'artemisApp.pages.tutorialGroupDetail.teachingAssistantNotSet' | artemisTranslate }}</span>
             </ng-template>
         </dd>
+        <!-- Utilization -->
+        <dt>
+            <span>{{ 'artemisApp.entities.tutorialGroup.utilization' | artemisTranslate }}</span>
+            <fa-icon [icon]="faQuestionCircle" class="text-secondary" ngbTooltip="{{ 'artemisApp.entities.tutorialGroup.utilizationHelp' | artemisTranslate }}"></fa-icon>
+        </dt>
+        <dd>
+            <ng-container *ngIf="tutorialGroup.averageAttendance !== undefined && tutorialGroup.capacity !== undefined; else noUtilization">
+                <jhi-vertical-progress-bar
+                    [fillLevelInPercent]="Math.round((tutorialGroup.averageAttendance / tutorialGroup.capacity) * 100)"
```

Comment:
Might be better to calculate this inside the component (computed property) and not the template.

Reply:
Personally I think this is really a matter of taste tbh.

Reply:
What do the other reviewers say? @simon-hng @bassner @Strohgelaender